README.md

```diff
@@ -8,9 +8,12 @@ Also provides functions to check if a [`S1-GUNW`](https://asf.alaska.edu/data-se
 
 ## Install
 
-`pip install .`
+1. Clone this repo `git clone https://github.com/ACCESS-Cloud-Based-InSAR/s1-enumerator.git`
+2. Navigate with your terminal to the repo.
+3. Create a new environment and install requirements using `conda env update -n s1-enumerator --file environment.yml` (or use [`mamba`](https://github.com/mamba-org/mamba) to speed install up)
+4. Install the package from cloned repo using `pip install .`
```

Comment:
Two minor comments:
* You shouldn't need the `-n s1-enumerator` with `conda env update` (you would for a `conda update`) as the name is in the yaml
* typically, when cloning the repo, you'll want to do `python -m pip install -e .` -- `python -m` is the preferred invocation now for pip (the `pip` script is going away) and `-e/--editable` means any changes you make locally will be immediately reflected in the environment